core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -1220,6 +1221,21 @@ class KafkaApis(val requestChannel: RequestChannel,
     }
   }
 
+  def handleDeleteGroupsRequest(request: RequestChannel.Request): Unit = {
+    val deleteGroupsRequest = request.body[DeleteGroupsRequest]
+    var groups = deleteGroupsRequest.groups.asScala.toSet
+
+    val (authorizedGroups, unauthorizedGroups) = groups.partition { group =>
+      authorize(request.session, Delete, new Resource(Group, group))
+    }
+
+    val groupDeletionResult = groupCoordinator.handleDeleteGroups(authorizedGroups)._2 ++
```

Comment:
looks like we are ignoring groupCoordinator.handleDeleteGroups(authorizedGroups)._1 error here.
handleDeleteGroups(authorizedGroups) can return Errors.COORDINATOR_NOT_AVAILABLE.

Reply:
Thanks for catching this. I missed updating this with the recent change to the protocol. Will update it in the next commit.